components/Card.js

```diff
@@ -1,7 +1,48 @@
 import React from "react";
+import Card from "react-bootstrap/Card";
 
-const Card = () => {
-	return <div>Card</div>;
-};
+class Block extends React.Component {
+	render() {
+		return (
+			// have one shadow white and one black right behind it looking like a border
+			<div className='flex justify-center'>
+				<Card
+					style={{
```

Comment:
Try to implement the styling using Tailwind

Reply:
That is using [tailwind ](https://tailwindcss.com/docs/justify-content)